### ISSUE TITLE
Cursor jumps to the first line from the end of the file

### ISSUE THREAD

**Issue by tamo** [hedgedoc/react-client#2446]
**Description**
If these three conditions are met;
1. another user's cursor is at the last line of a note,
2. the line is empty, and
3. your cursor is at the same position,

then pressing ↓-key will move your cursor to the first line.

**To Reproduce**
1. Create a note
2. Press Enter-key several times to make empty lines
3. Make sure your cursor is at the bottom of the file
4. Open the note with another browser window
5. See your cursor is at the top of the file (and the other at the bottom)
6. Press Down(↓)-key several times to move your cursor to the bottom of the file
7. Press it once more
8. See your cursor at the top

**Expected behavior**
Your cursor should stay at the bottom line.
(because it stays there unless another cursor is at the empty last line)

**Screenshots**

![image](https://user-images.githubusercontent.com/383537/196434331-8787ee57-7e1f-4100-9697-0152ba9f843a.png)

**Your Setup (please complete the following information):**
 - OS: Windows 10 (browser is native, nodejs is on wsl2 ubuntu)
 - Browser with version: Chrome 106 and Firefox 105
 - HedgeDoc version
     - server 2.0.0-dev
     - client CLIENT_VERSION_MISSING (package.json says it is 0.0.1)

**Additional context**
$ grep version node_modules/codemirror/package.json
  "version": "6.0.1",
$ grep version node_modules/@codemirror/view/package.json
  "version": "6.3.0",
$ grep version node_modules/@codemirror/state/package.json
  "version": "6.1.2",

**Comment by mrdrogdrog** [comment]
We can't change much here. The remote cursors are rendered by another lib. By https://github.com/yjs/y-codemirror to be precise.

**Comment by mrdrogdrog** [comment]
Even if not completly the same, this seems to be similar https://github.com/codemirror/dev/issues/800

**Comment by mrdrogdrog** [comment]
That's true. But the widgetBuffer is created by codemirror itself and there is no configuration option. So we can't influence it without either dirty dom hacks (pls not), changes in codemirror upstream or a codemirror forks.
Or am I missing something?
I think a ticket in the codemirror repository would be the best suggestion here.

**Comment by mrdrogdrog** [comment]
Maybe. I've been hoping that we could avoid forking other projects to apply patches that are only relevant for us.
The former devs of HedgeDoc 1 did that with CodeMirror 5 and that caused problems some weeks ago because the fork was horribly outdated :sweat: 

However. Let's see if we'll get an answer from the CodeMirror 6 devs.

**Comment by mrdrogdrog** [comment]
Thanks for your help with debugging this problem :heart:
